governance/polls/templates/Atlas/Atlas Edit Monthly Cycle Proposal - $date_MONTH_DD,_YYYY.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Atlas Edit Monthly Cycle Proposal - $date_MONTH_DD,_YYYY
+summary: $sentence_summary
+discussion_link: $discussion_link
+parameters:
+    input_format:
+        type: single-choice
+        abstain: [0]
+    victory_conditions:
+        - {
+            type: 'and',
+            conditions: [
+                { type : plurality },
+                { type : comparison, comparator : '>=', value: 10000 }
+            ]
+        }
+        - {type : default, value : 2 }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: $YYYY-MM-DDT16:00:00
+end_date: $YYYY-MM-DDT16:00:00
+---
+# Atlas Edit Monthly Cycle Proposal - $date_MONTH_DD,_YYYY
+
+The $poll_deployers have placed an [Atlas Edit Monthly Cycle Proposal](https://sky-atlas.powerhouse.io/#A.1.10.2_Atlas_Edit_Monthly_Cycle-430185a5-fa5d-4664-89cf-21c9e6cfc109|0db3326e) into the [voting system](https://vote.makerdao.com/polling). This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.2_Atlas_Edit_Weekly_Cycle-4a8ad9ad-5c5d-4994-9b46-f04c0e61ce59%7C0db30308) will be active for fourteen days beginning on $date_DAY,_MONTH_DD at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
```

Comment:
```suggestion

- **You may vote for a single option.**
```

Reply:
As per [this document](https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md032.md), "In addition to aesthetic reasons, some parsers, including kramdown, will not parse lists that don't have blank lines before and after them".